tests/integration/docker_images/popular-containers-images.bats

```diff
@@ -0,0 +1,405 @@
+	skip "this image is not working (see https://github.com/01org/cc-oci-runtime/issues/318)"
+	if $DOCKER_EXE run -i wordpress perl -e 'print "test\n"' | grep "LANG"; then false; else true; fi
+}
+
+@test "display the cqlsh version in a cassandra container" {
+	$DOCKER_EXE run --rm -i cassandra cqlsh --version
+}
+
+@test "cat a file in a centos container" {
+	$DOCKER_EXE run --rm -i centos bash -c "echo "Test" > testfile.txt | cat testfile.txt | grep "Test""
+}
+
+@test "run a couchbase container" {
+	skip "this image is not working (see https://github.com/01org/cc-oci-runtime/issues/318)"
+	$DOCKER_EXE run -i couchbase couchbase-cli rebalance --help | grep "rebalance"
+}
+
+@test "display options in a chronograf container" {
+	$DOCKER_EXE run --rm -i chronograf -sample-config | grep "configuration"
+}
+
+@test "run an instance in a traefik container" {
+	skip "this image is not working (see https://github.com/01org/cc-oci-runtime/issues/222)"
+	if $DOCKER_EXE run -i traefik traefik --version | grep "EXEC spawning"; then false; else true; fi
+}
+
+@test "run a consul container" {
+	$DOCKER_EXE run --rm -i consul timeout -t 10 consul agent -dev -client 0.0.0.0 | grep "0.0.0.0"
+}
+
+@test "run a gcc container" {
+	$DOCKER_EXE run --rm -i gcc bash -c "echo -e '#include<stdio.h>\nint main (void)\n{printf(\"Hello\");return 0;}' > demo.c && gcc demo.c -o demo && ./demo"
+}
+
+@test "run a crate container" {
+	$DOCKER_EXE run --rm -i -e CRATE_HEAP_SIZE=1g crate timeout -t 10 crate -h | grep "starts"
+}
+
+@test "check the resolv.conf in a crux container" {
+	$DOCKER_EXE run --rm -i crux bash -c "cat /etc/resolv.conf | grep "nameserver""
+}
+
+@test "find the timestamp help in a gazebo container" {
+	$DOCKER_EXE run --rm -i gazebo gz log --help
+}
+
+@test "go env in a golang container" {
+	$DOCKER_EXE run --rm -i golang bash -c "golang go env | grep "GOARCH""
+}
+
+@test "set memory size in haproxy container" {
+	$DOCKER_EXE run --rm -i haproxy haproxy -m 2 | grep "version"
+}
+
+@test "run an instance in an elixir container" {
+	$DOCKER_EXE run --rm -i elixir bash -c "pwd"
+}
+
+@test "run an odoo container" {
+	$DOCKER_EXE run --rm -i odoo bash -c "python -m timeit -s \"M=range(1000)\" \"L=[m*2 for m in M]\""
+}
+
+@test "start an instance of a mariadb container" {
+	$DOCKER_EXE run --rm -i -e MYSQL_ROOT_PASSWORD=secretword  mariadb bash -c "cat /etc/mysql/conf.d/mariadb.cnf | grep "character"" 
+}
+
+@test "run haskell container" {
+	$DOCKER_EXE run --rm -i haskell cabal --numeric-version
+}
+
+@test "run python command in a hylang container" {
+	$DOCKER_EXE run --rm -i hylang bash -c "python -m timeit -s \"L=[]; M=range(1000)\" \"for m in M: L.append(m*2)\""
+}
+
+@test "display config information of a influxdb container" {
+	$DOCKER_EXE run --rm -i influxdb influxd config | grep "dir"
+}
+
+@test "search nano in an ubuntu upstart container" {
+	$DOCKER_EXE run --rm -i ubuntu-upstart bash -c "apt-cache search nano | grep "editor""
```

Comment:
This test (and some of the other tests which run `grep`) look like they might be fragile - what if `LANG` is not set to a `C` locale and the command being run is internationalised?

A quick solution would be to add something like this to the top of `popular-containers-images.bats`:

```
export LANG=C
export LC_ALL=C
```